src/styles/App.css

```diff
@@ -0,0 +1,107 @@
+* {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+.MainBody {
+  width: 500px;
+  background: rgb(214, 126, 126);
+  display: grid;
+  grid-template-rows: auto auto auto;
+  gap: 1px;
+  margin: 1% 0 0 auto;
+}
```

Comment:
**- [ ]  [OPTIONAL] Well done completing the Math magician website :clap: , I think it would be better if we remove the whitespace and have text on the left side of the calculator so it can look more plesant like in the wireframe :point_down:** 

![image](https://user-images.githubusercontent.com/97521557/214773756-451b7f1a-f333-4467-912a-f2a1a909b732.png)